src/Altinn.Platform/Altinn.Platform.Receipt/Tests/ReceiptControllerTest.cs

```diff
@@ -0,0 +1,284 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Claims;
+
+using Altinn.Platform.Profile.Models;
+using Altinn.Platform.Receipt;
+using Altinn.Platform.Receipt.Helpers;
+using Altinn.Platform.Receipt.IntegrationTest.Mocks;
+using Altinn.Platform.Receipt.Model;
+using Altinn.Platform.Receipt.Services.Interfaces;
+using AltinnCore.Authentication.Constants;
+using AltinnCore.Authentication.JwtCookie;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+using Moq;
+using Newtonsoft.Json;
+using Tests.Mocks;
+using Tests.Testdata;
+using Xunit;
+
+namespace Test
```

Comment:
The base namespace for all classes in this project should be: `Altinn.Platform.Receipt.UnitTest`
Consider giving the project the same name.

Reply:
The thought was to have a single project for all tests and rather use partial classes to differ between unit and integration tests. I have renamed the project to include the altinn.platform.receipt prefisk